src/client/driver/driver.js

```diff
@@ -614,6 +618,9 @@ export default class Driver {
         else if (command.type === COMMAND_TYPE.getBrowserConsoleMessages)
             this._onGetBrowserConsoleMessagesCommand(command);
 
+        else if (command.type === COMMAND_TYPE.getBrowserInfo)
```

Comment:
Just curious why you decided to implement it as a client command? As far as I understand all the required info can be extracted from the user agent string. Can't this information be stored and provided by `hammerhead` as user agent string is available in the `User-Agent` header during the page request?

Reply:
Hello, @inikulin.

You are right, browser information will be presented as a `testController` property, based on the `user-agent` string. With this "draft/WIP" PR, I just wanted to see CI results. I will update it as soon as possible.